src/osd-overlay/worker.ts

```diff
@@ -114,11 +114,13 @@ export class VideoWorker {
   }
 
   modifyFrame(frame: ImageBitmap, frameIndex: number): ImageBitmap {
-    const osdCanvas = this.osdCanvas!;
-    const osdCtx = this.osdCtx!;
     const frameCanvas = this.frameCanvas!;
     const frameCtx = this.frameCtx!;
+    const osdCanvas = this.osdCanvas!;
```

Comment:
Why did you decide to change the order of initialization here?

Reply:
Seemed more logical since inside the function the frame is handled first followed by the osd